p2p/raintree/addrbook_utils_test.go

```diff
@@ -177,13 +181,13 @@ func testRainTreeMessageTargets(t *testing.T, expectedMsgProp *ExpectedRainTreeM
 	require.Equal(t, i, 0)
 
 	for _, target := range expectedMsgProp.targets {
-		addr, found := network.getFirstTargetAddr(uint32(target.level))
-		require.True(t, found)
-		require.Equal(t, addr, cryptoPocket.Address(target.left))
+		actualTargets := network.router.GetTargetsAtLevel(uint32(target.level))
 
-		addr, found = network.getSecondTargetAddr(uint32(target.level))
-		require.True(t, found)
-		require.Equal(t, addr, cryptoPocket.Address(target.right))
+		require.True(t, actualTargets[0].ShouldSendInternal())
```

Comment:
I think the unit tets that leverage `testRainTreeMessageTargets` should be split out into `router_test.go`